lib/beaker/host/unix/exec.rb

```diff
@@ -278,20 +278,14 @@ def clear_env_var key
   # @return [Result] result of restarting the SSH service
   def ssh_service_restart
     case self['platform']
-    when /debian|ubuntu/
-      exec(Beaker::Command.new("service ssh restart"))
-    when /amazon|(el|centos|redhat|oracle|scientific)-[7-9]|fedora|archlinux-/
-      exec(Beaker::Command.new("systemctl restart sshd.service"))
-    when /el-|centos|redhat|oracle|scientific/
+    when /(el|centos|redhat|oracle|scientific)-[0-6]/
       exec(Beaker::Command.new("/sbin/service sshd restart"))
-    when /opensuse|sles/
-      exec(Beaker::Command.new("/usr/sbin/rcsshd restart"))
```

Comment:
I think @joshcooper wanted to say that SLES 11 still hit this code. IMHO we should treat it like EL 6: special case it and less SLES 12+ fall through to systemd.

Reply:
Yes, I'm good with defaulting to systemd and special casing older platforms. Can you restore the ability to start services on SLES 11 using `/usr/sbin/rcsshd restart`?

Reply:
https://github.com/voxpupuli/beaker/pull/1883

Reply:
RHEL 6 will have Extended LIfecycle Support until June 30, 2024 based on https://access.redhat.com/articles/4665701#6-how-can-i-continue-to-receive-software-maintenance-bug-and-security-fixes-and-full-technical-support-for-red-hat-enterprise-linux-6-6

But they could extend it, similar to how Oracle extended Solaris 10 until Jan 2027 https://blogs.oracle.com/solaris/post/new-end-date-of-extended-support-for-oracle-solaris-10-and-113